vars/buildPlugin.groovy

```diff
@@ -115,7 +115,7 @@ def call(Map params = [:]) {
                                     infra.runMaven(mavenOptions, jdk, null, null, addToolEnv)
                                 } finally {
                                     if (!skipTests) {
-                                        junit('**/target/surefire-reports/**/*.xml,**/target/failsafe-reports/**/*.xml,**/target/invoker-reports/**/*.xml')
```

Comment:
If I understand correctly this change, the junit reports that would be in nested directories (for instance `target/surefire-reports/foo/test-1.xml`) would not be selected anymore.

Am I correct?

Reply:
I don't think they are actually nested

Reply:
surefire xml files are only generated at the top level directory.